### PR TITLE
Throw an error occurred inside Query.pluck()

### PR DESCRIPTION
**Pull request by tasanobu**
I fixed the issue reported in #414 to avoid app crash.

### CODE REVIEW

File: SQLite/Typed/Query.swift [hunk 1]
```diff
@@ -955,7 +955,10 @@ extension Connection {
     }
 
     public func pluck(query: QueryType) -> Row? {
-        return try! prepare(query.limit(1, query.clauses.limit?.offset)).generate().next()
+        guard let rows = try? prepare(query.limit(1, query.clauses.limit?.offset)) else {
+            return nil
```
Comment (nickmshelley) [inline SQLite/Typed/Query.swift]: We should probably propagate the error up instead of returning `nil`.

Comment (nickmshelley) [inline SQLite/Typed/Query.swift]: So basically I think this should be

``` swift
public func pluck(query: QueryType) throws -> Row? {
  return try prepare(query.limit(1, query.clauses.limit?.offset)).generate().next()
}

**Comment by tasanobu** [comment]
@nickmshelley
I’ve changed the implementation to propagate an error instead of returning nil

**Comment by nickmshelley** [comment]
Thanks @tasanobu!

@stephencelis I'd love to see this merged when you get some time.

**Pull request merged**